README.rst

```diff
@@ -143,3 +158,10 @@ which are the mandatory steps that users have to do to update to a new version,
 use but will not break any old code, and the **New Features** which are optional
 new features available in the new version that users might find interesting.
 Using them is optional, but encouraged.
+
+Contributing
+============
+
+See the guide for `contributing to Neptune-versioned libraries
+<https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst>`_.
```

Comment:
While this link covers details, I wonder if we should mention the `relnotes` folder at all in the README.

Perhaps add a paragraph to the end of the `Releases` section along the lines of:

```
Release notes are maintained in the `relnotes` folder of the source tree,
with individual files for each individual change.
```

Reply:
My feeling is that that's too much detail, at this level. The `Releases` section is about the final release in github (i.e. https://github.com/sociomantic-tsunami/ocean/releases). I don't think how those release notes are written is relevant here.

Reply:
(Happy to hear arguments to the contrary, though, if others feel differently :)